src/main/java/eu/yals/Endpoint.java

```diff
@@ -13,45 +10,148 @@ private Endpoint() {
+         * Application home page
+         */
+        public static final String HOME_PAGE = "";
+
+        /**
+         * Page that will appear, when user hits location which is not exist nor served by application
+         */
+        public static final String PAGE_404 = "errors/404";
+
+        /**
+         * Page that will shown when application hits server-side error
+         */
+        public static final String ERROR_PAGE_500 = "errors/500";
+
+        /**
+         * Page that will shown when application hits fatal error and became unaccessible
+         */
+        public static final String ERROR_PAGE_503 = "errors/503";
+
+        /**
+         * Page for debugging staff (avoid adding it in PROD mode)
+         */
+        public static final String DEBUG_PAGE = "debug";
+
+        /**
+         * Page for displaying software info (avoid adding it in PROD mode)
+         */
+        public static final String APP_INFO_PAGE = "appInfo";
+    }
+
+    /**
+     * API Endpoints
+     */
+    public static class Api {
+        /**
+         * API for storing links.
+         * <p>
+         * POST /api/store
+         */
+        public static final String STORE_API = "/api/store";
 
-    @RequestMethod(value = HttpMethod.GET, api = true)
-    public static final String LINK_API = "/api/link/";
-    public static final String LINK_API_MAPPING = "/api/link/{ident}";
-    public static final String LINK_API_BASE = "/api/link";
+        /**
+         * API for Getting links
+         * <p>
+         * GET /api/link/{ident}
+         */
+        public static final String LINK_API = "/api/link";
 
-    @RequestMethod(value = HttpMethod.POST, api = true)
-    public static final String MM_API = "/api/mm";
+        /**
+         * Mattermost API
+         * <p>
+         * POST /api/mm
+         */
+        public static final String MM_API = "/api/mm";
 
-    @RequestMethod(value = HttpMethod.GET, api = true)
-    public static final String TELEGRAM_STATUS_API = "/api/tg/status";
+        /**
+         * Show availability of Telegram API
+         * <p>
+         * GET /api/tg/status
+         */
+        public static final String TELEGRAM_STATUS_API = "/api/tg/status";
 
-    @RequestMethod(value = HttpMethod.GET, api = true)
-    public static final String QR_CODE_API_BASE = "/api/qrCode/";
-    public static final String QR_CODE_API = QR_CODE_API_BASE + "{ident}";
-    public static final String CUSTOM_SIZE_QR_CODE_API = QR_CODE_API + "/" + "{size}";
+        /**
+         * QR Code API
+         * <p>
+         * GET /api/qrCode/{ident}/{size}
+         */
+        public static final String QR_CODE_API = "/api/qrCode";
+
+        /**
+         * Page 404 for API requests
+         */
+        public static final String PAGE_404 = "/errors/404api";
+    }
 
-    @RequestMethod(HttpMethod.GET)
-    public static final String SLASH_BASE = "/";
-    public static final String SLASH = "/{ident}";
+    public static class Static {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Static' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)